apollo-router/src/plugins/connectors/tests.rs

```diff
@@ -452,13 +456,23 @@ async fn test_root_field_plus_entity_plus_requires() {
             "id": 1,
             "name": "Leanne Graham",
             "username": "Bret",
-            "d": "1-770-736-8031 x56442"
+            "d": "1-770-736-8031 x56442",
+            "rest": {
+              "phone": "1-770-736-8031 x56442",
+              "email": "Sincere@april.biz",
+              "website": "hildegard.org"
+            }
```

Comment:
Shouldn't this return a stringified version? Does GraphQL let you return objects for custom scalars?

Reply:
Custom scalars can be literally anything, as long as it can be serializable by the transport encoding!

I hadn’t noticed till now that the guild provides both a JSON scalar (stringified) and a JSONObject scalar (raw json values). 

I’m not sure how we’d support stringifying. `rest: *->stringify` maybe?

Reply:
oh nope i'm wrong. they have json for any json value, and jsonobject for specifically json objects: https://github.com/taion/graphql-type-json/blob/master/src/index.js. no JSON.parse() anywhere in the file